### DISCUSSION TITLE
chrome page not opening when using proxies

### DISCUSSION THREAD

**Discussion by warhawk200** [engageub/InternetIncome#179] [Q&A] [answered]
Hi , I have recently experienced that when i use direct connection then chrome page opens without any errors but when trying to open page with proxies enabled i get "ERR_CONNECTION_TIMED_OUT" 

i also tried opening same page with proxy-ip:7001 , but still get same results

without proxies it works ...

any idea , if and how this can be fixed ?

**Comment by engageub** [comment, marked as answer]
Hello,

While using proxies, please use serveo.net. Run the following command to
get url.


ssh -R 80:127.0.0.1:7000 serveo.net



More details here:
https://github.com/engageub/InternetIncome/wiki/Frequently-Asked-Questions#-how-to-access-127001-urls-for-ebesucher-or-mystnodes-if-gui-is-not-available-in-linux-

**Reply by warhawk200** [reply]
noice its worked , Thanks again

**Comment by warhawk200** [comment]
hello sir , i added 3 proxies and now when using servo command gives me this issue 
![image](https://github.com/user-attachments/assets/504e6b32-59e4-4354-8290-5b8373695d72)

**Reply by engageub** [reply]
Hello,
An alternative to serveo is localhost run.  Use this if serveo.net is busy.
Please use the following URL.  Replace 8080 with your port number,

```
ssh -R 80:127.0.0.1:8080 nokey@localhost.run
```

Thank you

**Reply by warhawk200** [reply]
Solved , Thanks boss
